WordingStatus.py

```diff
@@ -103,6 +104,8 @@ def load():
             Preferences.splitRegex         = re.compile(Preferences.splitRegex, re.U)
             Preferences.splitRegex         = Preferences.splitRegex.findall
 
+        Preferences.status_name            = sublime_settings.get('status_order_prefix', '')
```

Comment:
It is not prefixing as it is. This should solve this:
```suggestion
        Preferences.status_name            = sublime_settings.get('status_order_prefix', '') + 'WordCountStatus'
```

Reply:
Oops, silly mistake, sorry. Though I did test with empty prefixes, and it still worked, so apparently ST doesn't need a name for a status...